Lib/base64.py

```diff
@@ -108,29 +121,33 @@ def standard_b64decode(s):
 _urlsafe_encode_translation = bytes.maketrans(b'+/', b'-_')
 _urlsafe_decode_translation = bytes.maketrans(b'-_', b'+/')
 
-def urlsafe_b64encode(s):
+def urlsafe_b64encode(s, validate=False, padded=True):
```

Comment:
The `validate` arg isn't used. And if it's going to be added it should be added to `urlsafe_b64decode` instead.